.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: ruby
 rvm:
   - 2.1.0
+
+branches:
```

Comment:
That means we still need to update the list of "outdated" branches whenever a new one gets added, and I don't see how that would work really for travis - since it would mean we need to sync those changes to *all* branches that we care about.